{"name": "Попытка получения заказа с несуществующим id заказа", "status": "passed", "steps": [{"name": "Получение заказа по id заказа", "status": "passed", "parameters": [{"name": "order_id", "value": "0"}], "start": 1750763691434, "stop": 1750763691638}], "start": 1750763691433, "stop": 1750763691638, "uuid": "b920e972-eeb8-4a97-aa86-a6b6bde10864", "historyId": "a60899e78557f1db901f9fde6126f8d5", "testCaseId": "a60899e78557f1db901f9fde6126f8d5", "fullName": "tests.orders.test_get_order_with_id.TestGetOrderWithId#test_get_order_with_id_order_track_is_not_valid_error", "labels": [{"name": "feature", "value": "Получение заказа по id заказа"}, {"name": "parentSuite", "value": "tests.orders"}, {"name": "suite", "value": "test_get_order_with_id"}, {"name": "subSuite", "value": "TestGetOrderWithId"}, {"name": "host", "value": "LAPTOP-BQGT2AEK"}, {"name": "thread", "value": "18804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.orders.test_get_order_with_id"}]}